caluma/extensions/events/work_item.py

```diff
@@ -32,3 +32,39 @@ def create_circulation_child_case(sender, work_item, user, **kwargs):
             user=user,
             parent_work_item=work_item,
         )
+
+
+@on(post_create_work_item, raise_exception=True)
+@transaction.atomic
+def invite_to_circulation(sender, work_item, user, context, **kwargs):
+    if work_item.task_id == "circulation-decision" and context is not None:
+        for index, assign_user in enumerate(context["assign_users"]):
+            if not index:
+                work_item.assigned_users = [assign_user]
+                work_item.save()
+            else:
+                caluma_workflow_models.WorkItem.objects.create(
+                    name=work_item.task.name,
+                    description=work_item.task.description,
+                    task=work_item.task,
+                    status=caluma_workflow_models.WorkItem.STATUS_READY,
+                    assigned_users=[assign_user],
+                    case=work_item.case,
+                    document=caluma_form_models.Document.objects.create(
+                        form=work_item.document.form
+                    ),
+                )
```

Comment:
Changed the for to the suggestion, but the early return cant be implemented. If there is a context passed on a different workitem, it will also trigger

Reply:
Applied the change